scripts/convert_vtu_to_obj.py

```diff
@@ -0,0 +1,23 @@
+import meshio as mio 
+import sys
+
+def __main__():
```

Comment:
Isn't that something `meshio` can do? If not, having such a script is definitely nice, but please add a few lines of documentation, i.e. what does the script do, what are the arguments, and what is the output.

Reply:
meshio cannot convert mesh to obj with "line", I've added some docs